en_US/messages.json

```diff
@@ -208,5 +208,14 @@
     "neverOfferTranslation": {
       "message": "Never offer translations",
       "description": "Option to prevent the translation bar from being automatically displayed"
+    },
+    "translateWith": {
+      "message": "Translate with $FIREFOXTRANSLATIONS$",
+      "description":"Context menu item that copies the selected text in a popup dialog where it can be translated with Firefox Translations.",
+      "placeholders": {
+        "firefoxtranslations": {
+          "content": "Firefox Translations"
```

Comment:
This should be similar to the rest of the file

```
        "firefoxtranslations" : {
          "content": "$1",
          "example": "Firefox Translations"
        }
```

Reply:
why?

Reply:
I'm not the one making the format for WebExtensions
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Internationalization

Reply:
ok, that will represent one more allocation in the application side.